ignite/metrics/accuracy.py

```diff
@@ -14,11 +14,48 @@ class Accuracy(Metric):
     - `y_pred` must be in the following shape (batch_size, num_categories, ...) or (batch_size, ...)
     - `y` must be in the following shape (batch_size, ...)
     """
+    def __init__(self, is_multilabel=False, threshold_function=lambda x: torch.round(x), output_transform=lambda x: x):
+        super(Accuracy, self).__init__(output_transform)
+        self._threshold = threshold_function
+        if is_multilabel:
+            self.update = self._update_multilabel
```

Comment:
We also need to check whether self._threshold is callable in case of multi label.

Reply:
Will incorporate. added in latest commit.